src/components/search-by-demographics/search-by-demographics.component.tsx

```diff
@@ -0,0 +1,231 @@
+import React, {
+  Dispatch,
+  SetStateAction,
+  useCallback,
+  useEffect,
+  useState,
+} from "react";
+
+import {
+  DatePicker,
+  DatePickerInput,
+  Column,
+  NumberInput,
+  Switch,
+  ContentSwitcher,
+} from "carbon-components-react";
+import dayjs from "dayjs";
+import { useTranslation } from "react-i18next";
+
+import { SearchParams } from "../../types";
+import styles from "./search-by-demographics.style.scss";
+import {
+  getDescription,
+  getQueryDetails,
+} from "./search-by-demographics.utils";
+
+interface SearchByDemographicsProps {
+  setQueryDescription: Dispatch<SetStateAction<String>>;
+  setSearchParams: Dispatch<SetStateAction<SearchParams>>;
+  resetInputs: boolean;
+}
+
+export const SearchByDemographics: React.FC<SearchByDemographicsProps> = ({
+  setQueryDescription,
+  setSearchParams,
+  resetInputs,
+}) => {
+  const { t } = useTranslation();
+  const [livingStatus, setLivingStatus] = useState("alive");
```

Comment:
This should be, as follows, right? 

```suggestion
  const [livingStatus, setLivingStatus] = useState("alive" | "dead");
```